src/OpenTok/Archive.php

```diff
@@ -49,6 +49,10 @@
 * @property string $size
 * The size of the MP4 file. For archives that have not been generated, this value is set to 0.
 *
+* @property string $streamMode
+* Allows for an archive stream to be manually updated to add streams individually.
+* Default is set to <code>auto</code>.
```

Comment:
See the description for the `streamMode` option for the REST method for starting an archive: https://tokbox.com/developer/rest/#start_archive. Note that we need to add methods for adding and removing streams from archives (that use `streamMode` manual) to the PHP SDK (see this REST API method: https://tokbox.com/developer/rest/#selecting-archive-streams).

The same comment applies to the `streamMode` for broadcasts.

Reply:
These methods have now been added